documentation/asciidoc/computers/configuration/uart.adoc

```diff
@@ -171,7 +208,7 @@ By default, the primary UART is assigned to the Linux console. If you wish to us
 
 Although the Linux kernel starts the UARTs relatively early in the boot process, it is still long after some critical bits of infrastructure have been set up. A failure in those early stages can be hard to diagnose without access to the kernel log messages from that time. To enable `earlycon` support for one of the UARTs, add one of the following options to `cmdline.txt`, depending on which UART is the primary:
 
-For Raspberry Pi 4, 400 and Compute Module 4:
+For Raspberry Pi 5, 4, 400 and Compute Module 4:
```

Comment:
No - Pi 5 is different. The earlycon output only appears on the 3-pin Debug connector, and the magic runes are:
```
earlycon=pl011,0x107d001000,115200n8
```